examples/data-objects/table-document/src/document.ts

```diff
@@ -78,8 +78,7 @@ export class TableDocument extends DataObject<{ Events: ITableDocumentEvents }>
 	}
 
 	public async getRange(label: string) {
```

Comment:
Do we have test coverage if you break this function?

Assuming yes, let's avoid changing the public API of SparseMatrix: you already need to make a cast of something that isn't SharedString to a SharedString.

Instead, you should create the endpoint interval index inside the DataObject here and use `nextInterval` directly.

Reply:
if you wanted, you could change the signature of the index creation methods to only take in `SharedSegmentSequence` instead of `SharedString`, which would legitimize this call and still work, but I'm not sure the distinction is something we really want long-term considering we kind of want to deprecate all of the other shared sequences. So I think just doing the cast here is superior for now.

Reply:
If we don't have test coverage for this method breaking, you should add such a test in table-document as it's plausible we could at some point break it.

Reply:
I found there existed the test coverage on this method (but not tested directly). The method is invoked when the `tableDocument` creates a slice. Additionally, it requires the label to obtain the `intervalCollection`, which is not externally accessible. I believe it should be acceptable as long as incorporating the interval index does not break  the current tests.

Reply:
Yeah, if intentionally breaking this method causes existing tests to fail, I think we're fine. Thanks for checking!